src/game/ui/MainScreen.svelte

```diff
@@ -114,6 +115,8 @@
             <button class="big" on:click={() => layers = layers.next("connect")}>Connect to server</button>
             <div class="space"></div>
             <button on:click={() => layers = layers.next("settings")}>Settings</button>
+            <div class="space"></div>
+            <button on:click={() => quit()}>Quit</button>
```

Comment:
You could do this shorter:
```suggestion
            <button on:click={quit}>Quit</button>
```
Otherwise everything looks good!